shotgun_api3/lib/xmlrpclib.py

```diff
@@ -188,8 +189,8 @@ def _stringify(string):
 #__version__ = "1.0.1"
 
 # xmlrpc integer limits
-MAXINT =  2L**31-1
-MININT = -2L**31
+MAXINT =  2**31-1
+MININT = -2**31
```

Comment:
No impact: `2L` is strictly equivalent to `2` in py2 (it's a small int, the `L` was never needed)